ci/build-binaries-and-appimage.sh

```diff
@@ -15,8 +15,16 @@ export PATH=/deps/bin:"$PATH"
 echo "$KEY" | md5sum
 
 # we always build in a temporary directory
-# use RAM disk if possible
+# use RAM disk if possible and if enough space available
+USE_SHM=0
 if [ -d /dev/shm ] && mount | grep /dev/shm | grep -v -q noexec; then
+    SHM_FREE_KIB_MIN=$((1 * 1024 * 1024))
+    SHM_FREE_KIB=$(df -P -k /dev/shm | tail -n 1 | sed -e 's/ \+/ /g' | cut -d ' ' -f 4)
```

Comment:
There is no explanation why this amount has been chosen, just some number crunching. I don't think we have to build within a ramdisk in the CI. Other CI scripts just check whether `CI` is set.

Reply:
The reason I stick with using a ram disk is that this is already being done, and I wanted this change to not change how building with Docker works.  I think it might be better to check if `docker` is really `podman` ("podman" will appear in `--version`'s output), so that local dev builds with Docker behave the same as CI builds with Docker.  Is that OK?

Reply:
The issue with `/dev/shm` is that it is often too small to contain all the binaries in my experience in Docker, hece I removed this snippet from many build scripts again.

Let's just add a comment why we're checking for 1G of RAM.